kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/elyra-image.yaml

```diff
@@ -0,0 +1,25 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: elyra-notebook-image
+  annotations:
+    opendatahub.io/notebook-image-url:
+      "https://github.com/thoth-station/s2i-lab-elyra"
+    opendatahub.io/notebook-image-name:
+      "Elyra Notebook Image"
```

Comment:
If this is custom image, should we annotate that in the `notebook-image-name` ?

Reply:
What do you mean by custom image @erikerlandson? 

This is the odh official elyra image, there is also a custom-s2i-lab-elyra image, used to copy source content into the repo when building.

Reply:
I thought we already had a notebook menu option that was labeled 'Elyra Notebook Image' but if we do not, then this is fine.

Reply:
I see only two images linked to ArgoCD, is there some other image manually added?